include/aspect/particle/output/null.h

```diff
@@ -0,0 +1,78 @@
+/*
+ Copyright (C) 2011 - 2015 by the authors of the ASPECT code.
+
+ This file is part of ASPECT.
+
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file doc/COPYING.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef __aspect__particle_output_null_h
+#define __aspect__particle_output_null_h
+
+#include <aspect/particle/output/interface.h>
+
+
+namespace aspect
+{
+  namespace Particle
+  {
+    namespace Output
+    {
+      /**
+       * Blank class to avoid output of data, for example, if particles are used to represent
+       * other physical phenomenon that influences the simulation and we don't care about their positions
```

Comment:
Yes, I guess this was the initial idea. I did not think about the purpose of this class, simply took it from the former implementation and transferred it to the plugin system. I guess it would be possible to simply create no output plugin at all? In that case the tracer postprocessor would need to check though if an output plugin was created before calling `output->output_particle_data()`. Or could we merge this class into the interface class by giving it a default implementation which simply does not write output and then create an Interface class object?